torch/_inductor/utils.py

```diff
@@ -1069,8 +1072,8 @@ def use_cpp_packed_gemm_template(layout, mat1, mat2):
         m,
         n,
         k,
-        input_dtype=layout.dtype,
-        output_dtype=torch.float,
+        input_dtype=mat1.get_dtype() if int8_gemm else layout.dtype,
+        output_dtype=torch.int32 if int8_gemm else torch.float32,
```

Comment:
Not necessary in this PR but perhaps we can factor out a util function to return the compute and output dtype given the input dtype of micro gemm?

Reply:
Sure, will add the util function in the following PR